csrankings-m.csv

```diff
@@ -1148,6 +1145,8 @@ Matthew Green 0001,Johns Hopkins University,https://isi.jhu.edu/~mgreen,X0XWAGkA
 Matthew Guzdial,University of Alberta,http://guzdial.com,jKqmTbIAAAAJ
 Matthew Hague,Royal Holloway University of London,https://pure.royalholloway.ac.uk/portal/en/persons/matthew-hague(c4baa4a0-192e-49c8-a699-feda6a04a583).html,s37BfYAAAAAJ
 Matthew Hicks,Virginia Tech,http://www.impedimenttoprogress.com,_rJCgzIAAAAJ
+Matthew Hole,Australian National University,https://comp.anu.edu.au/people/matthew-hole/,BoPimVUAAAAJ

```

Comment:
Not listed as an alias in DBLP. If they are the same person, please send the DBLP folks a note and copy me.

Reply:
Email attached
[Untitled.pdf](https://github.com/emeryberger/CSrankings/files/8401463/Untitled.pdf)
.